app/views/vat/VatLanding.scala.html

```diff
@@ -0,0 +1,79 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
+@import controllers.routes
+@import views.html.templates.Layout
+@import messages.Messages
+@import requests.RequestSupport
+@import play.api.mvc.Request
+@import essttp.rootmodel.TaxRegime
+
+@this(
+        layout: Layout,
+        govukButton: GovukButton,
+        requestSupport: RequestSupport,
+        appConfig: AppConfig,
+        insetText: GovukInsetText
+)
+
+@()(implicit request: Request[_])
+
+@import requestSupport._
+
+@title= @{Messages.Vat.`Set up a VAT payment plan`.show}
+
+@layout(
+    pageTitle = Some(title),
+    showBackLink = false,
+    taxRegime = TaxRegime.Vat
+) {
+    <h1 class="govuk-heading-xl">@title</h1>
+    <p class="govuk-body">@{Messages.Vat.`The payment plan covers all appropriate overdue amounts...`.show}</p>
+    <h2 class="govuk-heading-m">@{Messages.Vat.`Who can use this service`.show}</h2>
+    <p class="govuk-body">@{Messages.Vat.`You are eligible to set up an online payment plan if:`.show}</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>@{Messages.Vat.`you plan to pay the debt off within the next 6 months or less`(appConfig.PolicyParameters.VAT.maxPlanDurationInMonths).show}</li>
+        <li>@{Messages.Vat.`you owe £20,000 or less`(appConfig.PolicyParameters.VAT.maxAmountOfDebt).show}</li>
+        <li>@{Messages.Vat.`you do not have any other payment plans or debts with HMRC`.show}</li>
+        <li>@{Messages.Vat.`your tax returns are up to date`.show}</li>
+    </ul>
+    <p class="govuk-body">@{Messages.Vat.`You can use this service within 28 days of the overdue payment deadline.`(appConfig.PolicyParameters.VAT.maxAgeOfDebtInDays).show}</p>
+
+    @insetText(InsetText(content = Text(Messages.Vat.`If you have a Customer Compliance Manager...`.show)))
+
+    <h2 class="govuk-heading-m">@{Messages.Vat.`Before you start`.show}</h2>
+    <p class="govuk-body">@{Messages.Vat.`You must be:`.show}</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>@{Messages.Vat.`a named account holder for the UK bank account you intend to use`.show}</li>
+        <li>@{Messages.Vat.`able to authorise a Direct Debit`.show}</li>
+    </ul>
+
+    <h2 class="govuk-heading-m">@{Messages.Vat.`Who cannot use this service:`.show}</h2>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>@{Messages.Vat.`cash accounting customers`.show}</li>
+        <li>@{Messages.Vat.`annual accounting scheme members`.show}</li>
+        <li>@{Messages.Vat.`payment on account customers`.show}</li>
+    </ul>
+
+    <p class="govuk-body">@{Messages.Epaye.`You must keep up to date with your payments...`.show}</p>
+
+    @govukButton(Button(
+        href = Some(routes.DetermineTaxIdController.determineTaxId.url),
+        content = Text(Messages.Epaye.`Start now`.show)
```

Comment:
`Start now` can be moved up a level out of `Epaye` I think since it's generic

Reply:
Ok